interop-testing/src/test/java/io/grpc/testing/integration/Http2OkHttpTest.java

```diff
@@ -88,8 +88,7 @@ protected ManagedChannel createChannel() {
         .overrideAuthority(GrpcUtil.authorityFromHostAndPort(
             TestUtils.TEST_SERVER_HOST, serverPort));
     try {
-      builder.sslSocketFactory(TestUtils.newSslSocketFactoryForCa(
-          TestUtils.loadCert("ca.pem")));
+      builder.sslSocketFactory(TestUtils.newSslSocketFactoryForCa(TestUtils.loadCert("ca.pem")));
```

Comment:
Oh, I follow, because swapping it would require you to start including the CA in the classpath. Makes sense to leave it.